_posts/2019-02-20-bitcoin-abc-0.19.0.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Prepare for the protocol upgrade today with 0.19.0
```

Comment:
Maybe should expand to:
"Prepare for the Bitcoin Cash protocol upgrade today with Bitcoin ABC 0.19.0"